### PR TITLE
Migrating Hosted Che e2e CI jobs from dev-cluster to OSD 4

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Migrating Hosted Che e2e CI jobs from dev-cluster to OSD 4

### What issues does this PR fix or reference?
https://github.com/redhat-developer/rh-che/issues/1798

### How have you tested this PR?
test via CI jobs

### CODE REVIEW

File: .ci/cico_build_deploy_test_rhche.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ else
   echo "Switching to project failed, probably not exists [$?]. Creating..."
   oc new-project "${PROJECT_NAMESPACE}" --display-name="PR ${RH_PULL_REQUEST_ID} - Automated Deployment" > /dev/null 2>&1
 fi
-oc policy add-role-to-user edit Katka92 ScrewTSW rhopp tomgeorge ibuziuk amisevsk davidfestal skabashnyuk -n $PROJECT_NAMESPACE
+oc policy add-role-to-user edit ibuziuk -n $PROJECT_NAMESPACE
```
Comment (ScrewTSW) [inline .ci/cico_build_deploy_test_rhche.sh]: One thing I don't understand is why are we being removed from the adminlist?

File: .ci/cico_rollout_test.sh [hunk 2]
```diff
@@ -114,7 +114,7 @@ oc login "${DEV_CLUSTER_URL}" --insecure-skip-tls-verify \
 echo "setting project"
 oc new-project "${PROJECT}" --display-name="PR ${RH_PULL_REQUEST_ID} - Automated Deployment" > /dev/null 2>&1
 echo "adding roles to project"
-oc policy add-role-to-user edit Katka92 ScrewTSW rhopp tomgeorge ibuziuk amisevsk davidfestal skabashnyuk -n $PROJECT
+oc policy add-role-to-user edit ibuziuk -n $PROJECT
```
Comment (ScrewTSW) [inline .ci/cico_rollout_test.sh]: Same question here.

**Comment by Katka92** [comment]
It seems that token used for log in to the OC cluster is not set properly:
```
error: The token provided is invalid or expired.
```

File: .ci/cico_build_deploy_test_rhche.sh (hunk 1 above)
Comment (ibuziuk) [reply .ci/cico_build_deploy_test_rhche.sh]: because I want to test this first with the minimal users, and othes would be added in a separate PR

**Comment by ibuziuk** [comment]
@Katka92 I think it is expected since the token is from the old cluster atm

File: .ci/cico_build_deploy_test_rhche.sh (hunk 1 above)
Comment (ScrewTSW) [reply .ci/cico_build_deploy_test_rhche.sh]: 👍

**Comment by ibuziuk** [comment]
[test]

**Pull request merged**